scalatest/src/main/scala/io/cucumber/core/options/CucumberSuiteOptionsParser.scala

```diff
@@ -0,0 +1,149 @@
+package io.cucumber.core.options
+
+import io.cucumber.core.exception.CucumberException
+import io.cucumber.core.feature.{FeatureWithLines, GluePath}
+import io.cucumber.core.resource.ClasspathSupport.CLASSPATH_SCHEME_PREFIX
+import io.cucumber.core.snippets.SnippetType
+import io.cucumber.scalatest.CucumberSuiteOptions
+import io.cucumber.tagexpressions.{TagExpressionException, TagExpressionParser}
+
+import java.util.regex.Pattern
+import scala.jdk.OptionConverters.RichOptional
+import scala.util.{Failure, Success, Try}
+
+object CucumberSuiteOptionsParser {
+
+  def unsafeParse[T <: CucumberSuiteOptions](
+      options: T
+  ): RuntimeOptionsBuilder = {
+    parse(options) match {
+      case Success(value) => value
+      case Failure(exception) =>
+        println(
```

Comment:
are we sure that we would like to have this print in SOUT asis ?

Reply:
Probably not but so far I just got inspired from the cucumber-junit module and copy/pasted stuff from there.